package.json

```diff
@@ -300,7 +300,7 @@
     "@metamask/browser-passworder": "^4.3.0",
     "@metamask/contract-metadata": "^2.5.0",
     "@metamask/controller-utils": "^11.4.0",
-    "@metamask/design-tokens": "^4.2.0",
+    "@metamask/design-tokens": "^5.0.0",
```

Comment:
Upgrading [design tokens to v5](https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/CHANGELOG.md#500)